### PR TITLE
Resolve issue #18

### LINKED ISSUE #18: Arrow for implication
Great package!

Is it possible to add  `→` (\rightarrow<tab>) or something similar for implication? It's a bit cleaner than `-->`

### PR DESCRIPTION
**Pull request by AugustoCL**
To resolve #18, this PR add the new operator → (`\to<tab>` or `\rightarrow<tab>`) by adding:
- Code and docstring in `macros.jl`
- The operator in `README.md`
- Tests in `runtests.jl`
- The export of the operator in `TruthTables.jl`

### CODE REVIEW

**Comment by eliascarv** [comment]
@AugustoCL please merge main into your branch.

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ This is the list of symbols that can be used:
 | XOR  | `⊻` (`\xor<tab>`) |
 | NAND | `⊼` (`\nand<tab>`) |
 | NOR  | `⊽` (`\nor<tab>`) |
-| IMPLICATION | `-->` |
+| IMPLICATION | `-->`, `→` (`\to<tab>` or `\rightarrow<tab>`) |
 | EQUIVALENCE | `<-->`, `≡` (`\equiv<tab>`) |
```
Comment (eliascarv) [inline README.md]: Add the other operators in README and docstring.

**Comment by AugustoCL** [comment]
After commit 2f71564, this PR adds 4 new unicode operators:

→ (`\to` or `\rightarrow`)
↔ (`\leftrightarrow`)
⇒ (`\Rightarrow`)
⇔ (`\Leftrightarrow`)

File: README.md (hunk 1 above)
Comment (AugustoCL) [reply README.md]: Done in 5ac641b

**Pull request merged**